velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -215,7 +215,10 @@ std::unique_ptr<SplitReader> HiveDataSource::createSplitReader() {
       ioStats_,
       fileHandleFactory_,
       executor_,
-      scanSpec_);
+      scanSpec_,
+      remainingFilterExprSet_,
```

Comment:
You are not really using this inside `IcebergSplitReader` (instead you have your own `deleteExprSet_`), let's avoid sharing and passing it.